webapp/security/views.py

```diff
@@ -249,6 +249,7 @@ def _update_notice_object(notice, data):
     notice.published = data["published"]
     notice.references = data["references"]
     notice.instructions = data["instructions"]
+    notice.is_hidden = False
```

Comment:
Could we not just do `notice.is_hidden = data.get("is_hidden", False)` and remove the if?

Reply:
Changed!